DataFormats/L1Trigger/src/L1PFTau.cc

```diff
@@ -10,7 +10,7 @@ using std::dec;
 typedef std::vector<l1t::L1PFTau> L1PFTauCollection;
 
 // default constructor
-l1t::L1PFTau::L1PFTau() : m_data(0), m_tauType(12), m_tauIsoQual(0), m_tauRelIsoQual(0), m_relativeIsolation(100), m_rawIsolation(0), m_chargedIsolation(0), m_passTightIso(0), m_passMediumIso(0),m_passLooseIso(0), m_passVLooseIso(0), m_passTightRelIso(0), m_passMediumRelIso(0),m_passLooseRelIso(0), m_passVLooseRelIso(0){  };
+l1t::L1PFTau::L1PFTau() : m_data(0), m_tauType(12), m_tauIsoQual(0), m_tauRelIsoQual(0), m_relativeIsolation(100), m_rawIsolation(0), m_chargedIsolation(0), m_passTightIso(0), m_passMediumIso(0),m_passLooseIso(0), m_passVLooseIso(0), m_passTightRelIso(0), m_passMediumRelIso(0),m_passLooseRelIso(0), m_passVLooseRelIso(0), m_EoH(0), m_HoE(0){  };
```

Comment:
Need to obey the order of assingnment in the default contructor for this code to compile. Please move `, m_EoH(0), m_HoE(0)` to after `m_relativeIsolation(100)`. 

This line should read
`l1t::L1PFTau::L1PFTau() : m_data(0), m_tauType(12), m_tauIsoQual(0), m_tauRelIsoQual(0), m_relativeIsolation(100), m_EoH(0), m_HoE(0), m_rawIsolation(0), m_chargedIsolation(0), m_passTightIso(0), m_passMediumIso(0),m_passLooseIso(0), m_passVLooseIso(0), m_passTightRelIso(0), m_passMediumRelIso(0),m_passLooseRelIso(0), m_passVLooseRelIso(0){  };`

Reply:
Since this is a trivial change,  I will fix it when merging.